cmd/forwarder/fixer_test.go

```diff
@@ -216,17 +208,22 @@ func httpRequestWithParams() *http.Request {
 	return req
 }
 
-func httpRequestWithCustomParams() *http.Request {
+func httpRequestWithFieldParams() *http.Request {
 	req, _ := http.NewRequest("POST", "/logs?index=i&source=s&sourcetype=st&custom1=cq1&custom2=cq2", nil)
 	return req
 }
 
-func httpRequestWithMetricsDestination() *http.Request {
-	req, _ := http.NewRequest("POST", "/logs?index=i&source=s&sourcetype=st&metrics-destination=argus,librato", nil)
-	return req
-}
-
 func simpleHttpRequest() *http.Request {
 	req, _ := http.NewRequest("POST", "/logs", nil)
 	return req
 }
+
+func getConfig() *IssConfig {
```

Comment:
Nit: I think that unless you're dealing with a large amount of data for copying, or you actually need *the same object*, general "recommended practice" in go is to avoid pointers and just copy the struct. ([source](https://medium.com/@meeusdylan/when-to-use-pointers-in-go-44c15fe04eac)).

Reply:
That's interesting.  I was going by [this](https://github.com/golang/go/wiki/CodeReviewComments#receiver-type) which says when in doubt use a pointer.  I'll profile and see which is the better performance then update accordingly.